### ISSUE TITLE
Throws error while used in Interfacebuilder

### ISSUE THREAD

**Issue by Iniyarajan** [Skyscanner/SkyFloatingLabelTextField#132]
Unknown class _TtC33SkyFloatingLabelTextFieldWithIcon33SkyFloatingLabelTextFieldWithIcon in Interface Builder file.

**Comment by k0nserv** [comment]
Hey,

To better understand what's going wrong I need to know how are you integrarting SkyFloatingLabelTextField in your project?

**Comment by Iniyarajan** [comment]
Initially I added by cocoa pods, On doing it not detect the headers
So I downloaded the source file and added inside the project folder.
Now it works for SkyFloatingLabelTextField, and it crashes while using  SkyFloatingLabelTextFieldWithIcon.
May I know , what is correct flow of using this class ? 
What I'm doing wrong ?
Thanks,
Iniyarajan
![screen shot 2017-04-26 at 2 55 28 pm](https://cloud.githubusercontent.com/assets/19946775/25427567/a029afec-2a90-11e7-94c0-9ded5b1bde7c.png)

**Comment by k0nserv** [comment]
When you used CocoaPods did you remember to open `<Project>.xcworkspace` after running `pod install` instead of opening `<Project>.xcproject`? I'd recommend you stick with CocoaPods it'll be simpler for you in the long run.

I see in your screenshot that you now have both pods and the source files integrated which could create problems in itsel. What's the full error when it crashes?

**Comment by Iniyarajan** [comment]
Yes, I used the xcworkspace, But It shows error as below screenshot.
![screen shot 2017-04-26 at 3 09 48 pm](https://cloud.githubusercontent.com/assets/19946775/25428112/7485d878-2a92-11e7-87a7-6389dc78e39c.png)

**Comment by k0nserv** [comment]
I don't see an import statement for SkyFloatingLabelTextField in there. If you add

```
@import SkyFloatingLabelTextField
```

Does that solve it?

**Comment by Iniyarajan** [comment]
Now, I added and the error as below,
![screen shot 2017-04-26 at 3 14 50 pm](https://cloud.githubusercontent.com/assets/19946775/25428340/2fd8cb76-2a93-11e7-8c16-8df3a8c727a5.png)
![screen shot 2017-04-26 at 3 14 08 pm](https://cloud.githubusercontent.com/assets/19946775/25428347/3361f678-2a93-11e7-952d-4e0529360c31.png)

**Comment by k0nserv** [comment]
Are you sure you've connected the outlet correctly that looks like the value is `nil`. I'd put a breakpoint and verify that the value gets set correctly

**Comment by Iniyarajan** [comment]
Yes, i checked all the outlets are connected correctly.

**Comment by k0nserv** [comment]
There are a bunch of other things that could be wrong. Did you specify the module when setting the class in IB? 

I made a small example project that integrates SkyFloatingLabelTextField using CocoaPods and then connects a SkyFloatingLabelTextFieldWithIcon via IB like you are.
[TestIssue132.zip](https://github.com/Skyscanner/SkyFloatingLabelTextField/files/957978/TestIssue132.zip)

**Comment by k0nserv** [comment]
@Iniyarajan did you resolve this?
